src/components/FoodSearchComponent/FoodSearchComponent.jsx

```diff
@@ -18,7 +18,7 @@ class FoodSearchComponent extends Component {
     this.props.dispatchFoodSearch(this.state);
```

Comment:
this component will need a checkbox function to add multiple lines of food.  can be added to debug list.